doc/external-libs/trilinos.html

```diff
@@ -55,36 +55,53 @@
     </p>
 
     <p>
+      deal.II uses the following libraries from Trilinos and will fail to
+      compile with Trilinos if they are not present:
+      <ul>
+        <li> Amesos,
+        <li> AztecOO,
+        <li> Epetra,
+        <li> Ifpack,
+        <li> ML,
+        <li> MueLu (if using Trilinos 11.14 or newer),
+        <li> Sacado, and
+        <li> Teuchos.
+      </ul>
+
       Trilinos uses <a href="http://cmake.org/">cmake</a> to configure and
       build. The following slightly longish set of commands will set up a
       reasonable configuration (we require MueLu starting from 12.0):
       <pre>
 
-	cd trilinos-12.4.2
-	mkdir build
-	cd build
-
-	cmake \
-	-D Trilinos_ENABLE_Sacado=ON \
-	-D Trilinos_ENABLE_MueLu:BOOL=ON \
-	-D Trilinos_ENABLE_Stratimikos=ON \
-	-D CMAKE_BUILD_TYPE=RELEASE \
```

Comment:
i would keep it, not sure what is the default. Also officially it is called `Release` (https://cmake.org/cmake/help/v3.0/variable/CMAKE_BUILD_TYPE.html)